tests/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+find_package(GTest REQUIRED)
+
+set(TEST_NAME liblog_test)
```

Comment:
test targets names should be prefixed by the project name https://github.com/EVerest/libevse-security